src/scala/tiefvision-web/app/core/ImageProcessing.scala

```diff
@@ -24,8 +24,8 @@ object ImageProcessing {
   def randomSmilarityImage = SimilarityImages(random.nextInt(SimilarityImages.length))
 
   def findSimilarImages(image: String, finderFolder: String, imagesFolderOpt: Option[String] = None): ImageSearchResult = {
-    val imagesFolderOptionOpt = imagesFolderOpt.map(imagesFolder => s"-imagesFolder ${imagesFolder}").getOrElse("")
-    val finderProcessBuilder = Process(Seq("bash", "-c", s"luajit search.lua $imagesFolderOptionOpt -image $image"), new File(finderFolder))
+    val luaSearch = s"luajit search.lua $image ${imagesFolderOpt.getOrElse("")}"
```

Comment:
We have 3 different search scripts. Two of them search in a "database", and the last one compares an image to those found in a folder. 

The extra argument, `imagesFolderOpt`, is only relevant / used in this last case.